collect_app/src/main/java/org/odk/collect/android/map/MapboxUtils.java

```diff
@@ -0,0 +1,48 @@
+package org.odk.collect.android.map;
+
+import android.content.Context;
+import android.os.Build;
+import android.widget.Toast;
+
+import com.mapbox.mapboxsdk.Mapbox;
+
+import org.odk.collect.android.BuildConfig;
+import org.odk.collect.android.R;
+import org.odk.collect.android.application.Collect;
+
+public class MapboxUtils {
+    private static boolean initAttempted = false;
+    private static Mapbox mapbox = null;
```

Comment:
`pmd` complains about redundant initializers. I tend to like the explicitness so I'd be ok turning that check off.

It also recommends a private constructor which is reasonable.